ECL/Testmini.ecl

```diff
@@ -0,0 +1,138 @@
+IMPORT Python3 AS Python;
+IMPORT Std.System.Thorlib;
+anomaly:= $.minids.mini_file;
```

Comment:
Cannot find minids.mini_file. Please upload the datasets in the Datasets folder. Thank you.